### DISCUSSION TITLE
Concurrent document conversions slows down individual conversion

### DISCUSSION THREAD

**Discussion by blackheaven** [gotenberg/gotenberg#535] [Q&A] [answered]
I was trying to benchmark gotenberg v7.7.0 with `hyperfine` converting DOCX/PPT/XLS file (~1 MB each).

While running one benchmark at a time take 1.1s, two benchmarks take 7s, and three take 27s.

**Comment by gulien** [comment, marked as answer]
Hello @blackheaven,

By default, Gotenberg uses one and only one LibreOffice instance, and it can convert only one document at a time.

By using `—uno-listener-restart-threshold=0`, Gotenberg will start a dedicated LibreOffice instance per conversion, which is somewhat better for concurrent conversions, to the detriment of unitary conversion performance.
